packages/core/phase2/lib/generateOperations/resultClassHandlers/handleSentence.ts

```diff
@@ -23,18 +21,6 @@ export const handleSentence: ResultClassHandler = ({
     return { operations: [createOperation(PncOperation.PENALTY_HEARING, operationData)], exceptions: [] }
   }
 
-  if (!result.PNCAdjudicationExists) {
-    if (!offence.AddedByTheCourt) {
-      const exception = {
-        code: ExceptionCode.HO200106,
-        path: errorPaths.offence(offenceIndex).result(resultIndex).resultClass
-      }
-      return { operations: [], exceptions: [exception] }
-    }
-
-    return { operations: [], exceptions: [] }
```

Comment:
Removing this return changes the behaviour of the application. When `PNCAdjudicationExists` is `false` we should return and exit this function.
Since we don't want to generate the exception here, we should just return empty operations and exceptions.